lib/html5ever/native.ex

```diff
@@ -1,9 +1,43 @@
 defmodule Html5ever.Native do
   @moduledoc false
+  require Logger
+
+  mix_config = Mix.Project.config()
+  @version mix_config[:version]
+  # @github_url mix_config[:package][:links]["GitHub"]
+
+  rustler_opts = [otp_app: :html5ever, crate: "html5ever_nif", mode: :release]
+
+  opts =
+    if System.get_env("HTML5EVER_BUILD") in ["1", "true"] do
+      rustler_opts
+    else
+      case Html5ever.Precompiled.download_or_reuse_nif_file(
+             rustler_opts,
+             # TODO: change to the following before merging PR
+             # base_url: "#{@github_url}/releases/download/v#{@version}",
+             base_url:
+               "https://github.com/philss/html5ever_elixir/releases/download/testing-release30",
+             version: @version
+           ) do
+        {:ok, new_opts} ->
+          new_opts
+
+        {:error, error} ->
+          error = "Error while downloading precompiled NIF: #{inspect(error)}"
+
+          if Mix.env() == :prod and match?({:unix, _}, :os.type()) do
```

Comment:
Why the OS check? :)

Reply:
I wasn't raising the error for Windows because I didn't have the files. I can remove this now :)